byron/ledger/formal-spec/blockchain-interface.tex

```diff
@@ -17,9 +17,10 @@ \subsection{Delegation interface}
     \left(
       \begin{array}{r@{~\in~}lr}
         \mathcal{K} & \powerset{\VKeyGen} & \text{allowed delegators}\\
-        \var{e} & \Epoch & \text{epoch}\\
-        \var{s} & \Slot & \text{slot}\\
-        \var{pps} & \PPMMap & \text{Protocol parameters}
+        k & \mathbb{N} & \text{chain stability parameter}\\
```

Comment:
Again, I feel much less keen on these "env" parameters. What's the advantage of this?